app/package.json

```diff
@@ -28,15 +30,21 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-icons": "^4.3.1",
-    "styled-components": "^5.3.3"
+    "styled-components": "^5.3.3",
+    "ts-node": "^10.4.0"
   },
   "devDependencies": {
-    "@types/node": "16.11.6",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@types/jest": "^27.0.3",
+    "@types/node": "^16.11.6",
     "@types/react": "17.0.34",
     "eslint": "7.32.0",
     "eslint-config-next": "12.0.3",
     "husky": "^4.3.0",
+    "jest": "^27.4.5",
     "prettier": "^2.4.1",
-    "typescript": "4.4.4"
+    "ts-jest": "^27.1.2",
+    "typescript": "^4.4.4"
```

Comment:
We should specify an LTS version that we will share between all the project, especially for the jest node ts

Reply:
yep guess this one is the best in term of documentation / community